src/utils/util.ts

```diff
@@ -17,28 +16,28 @@ export const getHostname = () => {
 export const isDataEqual = <T>(
   oldData:
     | T
-    | List<T>
-    | List<List<T>>
+    | T[]
+    | T[][]
     // necessary for download avatar, thumbnail
     // might be removed if we only use links
     | Blob
     | undefined,
   newData:
     | T
-    | List<T>
-    | List<List<T>>
+    | T[]
+    | T[][]
     // necessary for download avatar, thumbnail
     // might be removed if we only use links
     | Blob,
-): boolean => is(oldData, newData);
+): boolean => oldData === newData; // TODO !!!!!!!!!!!!
```

Comment:
Do we need this function ? I think that react-query already is able to perform the comparison (deeply) with JS objects. We could just get rid of this one.